sdks/python/apache_beam/runners/common.py

```diff
@@ -449,7 +452,8 @@ def __init__(self,
                step_name=None,
                logging_context=None,
                state=None,
-               scoped_metrics_container=None):
+               scoped_metrics_container=None,
+               operation_name=None):
```

Comment:
What is the difference between `step_name` and `operation_name`?

Also, please document any additional arguments in the docstring below

Reply:
From my understanding so far, we use operation_name referred to external step(eg. original graph step) and use step_name referred to internal step(eg. optimized graph step). When we initialize a operation, operation.operation_name = operation.step_name, but will change step_name when calling create_operation.